packages/cli/pr-multi-file-drag-drop.md

```diff
@@ -0,0 +1,25 @@
+## Summary
```

Comment:
Can we move this file to the docs folder?

Seems like we can also drop the pr prefix.